capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisFragmentImpl.java

```diff
@@ -283,14 +282,21 @@ private void handleOnBackPressed() {
         activity.getOnBackPressedDispatcher().addCallback(mFragment.getViewLifecycleOwner(), new OnBackPressedCallback(true) {
             @Override
             public void handleOnBackPressed() {
-                trackAnalysisScreenEvent(AnalysisScreenEvent.CANCEL);
                 setEnabled(false);
                 remove();
-                mFragment.getActivity().getOnBackPressedDispatcher().onBackPressed();
+                onBack();
```

Comment:
[Request] Here as well we should only track the cancel event:
```suggestion
                trackAnalysisScreenEvent(AnalysisScreenEvent.CANCEL);
                setEnabled(false);
                remove();
```

Reply:
The same comment as for `CameraFragmentImpl`.

Reply:
Oh, I see! Then no changes needed. 👍